packages/cli/src/lib/bundler/config.ts

```diff
@@ -110,6 +110,12 @@ export async function createConfig(
     );
   }
 
+  plugins.push(
+    new ProvidePlugin({
+      process: 'process/browser',
```

Comment:
What's the story behind this one, maybe a comment? :>

Reply:
Because `process` is a global not like a `import process from 'process'` polyfill. Will add a comment.